lighthouse-core/audits/seo/font-size.js

```diff
@@ -293,7 +296,7 @@ class FontSize extends Audit {
         (failingTextLength - analyzedFailingTextLength) / visitedTextLength * 100;
 
       tableData.push({
-        source: 'Add\'l illegible text',
+        source: {type: 'code', value: 'Add\'l illegible text'},
```

Comment:
in case anyone wonders why these get overridden with a code type... its for this:

![image](https://user-images.githubusercontent.com/39191/63970512-344ef280-ca59-11e9-9e8d-89011c6aaad4.png)

Reply:
want this to be `text` now (#9626)?

Reply:
maybe add a comment about the default (`source-location`) and this is overriding that